GitVersion.yml

```diff
@@ -0,0 +1,12 @@
+next-version: 2.0.0
+mode: mainline 
```

Comment:
Can this be simplified to just use `mode: mainline` without all the other stuff?

Reply:
File cut down to just `mode: mainline`